docs/0 - install.md

```diff
@@ -50,6 +52,17 @@ table to store configuration options. As such, you should run the migrations.
 > php spark migrate --all
 ```
 
+---
+
+#### Note: migration error
+
+When you run `spark migrate --all`, if you get `Class "SQLite3" not found` error:
+
+1. Remove sample migration files in `tests/_support/Database/Migrations/`
```

Comment:
I don't see those files in this library. Did I miss something?

Reply:
Oh you mean in the implementing project! Got it.

Reply:
Yes!
These files cause the error.
- https://github.com/codeigniter4/appstarter/tree/master/tests/_support/Database/Migrations
- https://github.com/codeigniter4/appstarter/blob/a6f65a777efdc44b89e9bee835a8e5b703833f24/app/Config/Database.php#L65

Reply:
I responded in the other thread but I think we should remove these if they are causing more harm than good. Doesn't affect this PR though

Reply:
I don't know causing more harm than good.
It is a default sample test code. So if devs sees it, they can learn how to use.
And the error happens only in environment with no SQLite3 extension.

Reply:
Maybe the problem is that `migrate --all` is running migrations from the `Tests` namespace, which logically should never happen (and could cause a lot of problems if test migration files were present on a production instance).